python/private/py_wheel.bzl

```diff
@@ -29,6 +29,23 @@ PyWheelInfo = provider(
     },
 )
 
+PyRequirementInfo = provider(
+    doc = "Information about an individual python requirement",
+    fields = {
+        "name": "String: the name of the pypi dependency (e.g. 'protobuf')",
+        "specifier": "String: a formatted string, to be used verbatim for 'Requires-Dist' metadata specification (e.g 'protobuf>=4.25.2', see https://pip.pypa.io/en/stable/reference/requirement-specifiers/)",
+        "version": "String: the concrete version of the pypi requirement (e.g. '4.25.2')",
```

Comment:
Yes it is redundant.  I left `version` in because I thought *someone* might want to know the concrete version rather than having to parse the specifier, but since I don't have a specific use for it myself, should probably be removed for the sake of simplicity (could be added later)